hypnos/src/gfx.c

```diff
@@ -74,9 +73,14 @@ void gfx_date_set_label(char *str)
 
 void gfx_bt_set_label(int symbol)
 {
-	if (symbol) {
+	switch (symbol) {
+	case 1:
```

Comment:
usually use an Enum for this kind of cases to abstract it. This way you don't need to earch for the meaning of what is `1` or `2`

Reply:
Thanks! I've added enums in a separate commit.